traffic_portal/app/src/common/api/ServerService.js

```diff
@@ -42,6 +42,12 @@ var ServerService = function($http, locationUtils, messageModel, ENV) {
     };
 
     this.createServer = function(server) {
+        if (server.ip6Address === undefined || server.ip6Address === '') {
```

Comment:
`if (!server.ip6Address || server.ip6Address === '')` will check if ip6Address is undefined, null or blank

Reply:
actually, i think you can just do `if (!server.ip6Address)`

falsy values are null, undefined, empty strings, false, 0

so if it's any of those things, it will evaluate to true

Reply:
done

Reply:
You could just do `if (server.ip6Address)`:
```javascript
if ("") {console.log("true")} else {console.log("false")}
// prints "false"
```